Performing C SOURCE FILE Test CMAKE_HAVE_LIBC_PTHREAD failed with the following output:
Change Dir: /home/cdesio/TAT/tat_simu_2steps/simulation/build/CMakeFiles/CMakeTmp

Run Build Command(s):/usr/bin/gmake -f Makefile cmTC_c288c/fast && /usr/bin/gmake  -f CMakeFiles/cmTC_c288c.dir/build.make CMakeFiles/cmTC_c288c.dir/build
gmake[1]: Entering directory `/home/cdesio/TAT/tat_simu_2steps/simulation/build/CMakeFiles/CMakeTmp'
Building C object CMakeFiles/cmTC_c288c.dir/src.c.o
/opt/anaconda3/envs/rootpy/bin/x86_64-conda-linux-gnu-cc -DCMAKE_HAVE_LIBC_PTHREAD  -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /opt/anaconda3/envs/rootpy/include  -o CMakeFiles/cmTC_c288c.dir/src.c.o -c /home/cdesio/TAT/tat_simu_2steps/simulation/build/CMakeFiles/CMakeTmp/src.c
Linking C executable cmTC_c288c
/usr/local/bin/cmake -E cmake_link_script CMakeFiles/cmTC_c288c.dir/link.txt --verbose=1
/opt/anaconda3/envs/rootpy/bin/x86_64-conda-linux-gnu-cc -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /opt/anaconda3/envs/rootpy/include  -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,--allow-shlib-undefined -Wl,-rpath,/opt/anaconda3/envs/rootpy/lib -Wl,-rpath-link,/opt/anaconda3/envs/rootpy/lib -L/opt/anaconda3/envs/rootpy/lib  CMakeFiles/cmTC_c288c.dir/src.c.o -o cmTC_c288c 
/opt/anaconda3/envs/rootpy/bin/../lib/gcc/x86_64-conda-linux-gnu/10.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: CMakeFiles/cmTC_c288c.dir/src.c.o: in function `main':
src.c:(.text.startup.main+0x24): undefined reference to `pthread_create'
/opt/anaconda3/envs/rootpy/bin/../lib/gcc/x86_64-conda-linux-gnu/10.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: src.c:(.text.startup.main+0x2e): undefined reference to `pthread_detach'
/opt/anaconda3/envs/rootpy/bin/../lib/gcc/x86_64-conda-linux-gnu/10.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: src.c:(.text.startup.main+0x38): undefined reference to `pthread_cancel'
/opt/anaconda3/envs/rootpy/bin/../lib/gcc/x86_64-conda-linux-gnu/10.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: src.c:(.text.startup.main+0x44): undefined reference to `pthread_join'
/opt/anaconda3/envs/rootpy/bin/../lib/gcc/x86_64-conda-linux-gnu/10.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: src.c:(.text.startup.main+0x50): undefined reference to `pthread_atfork'
collect2: error: ld returned 1 exit status
gmake[1]: *** [cmTC_c288c] Error 1
gmake[1]: Leaving directory `/home/cdesio/TAT/tat_simu_2steps/simulation/build/CMakeFiles/CMakeTmp'
gmake: *** [cmTC_c288c/fast] Error 2


Source file was:
#include <pthread.h>

static void* test_func(void* data)
{
  return data;
}

int main(void)
{
  pthread_t thread;
  pthread_create(&thread, NULL, test_func, NULL);
  pthread_detach(thread);
  pthread_cancel(thread);
  pthread_join(thread, NULL);
  pthread_atfork(NULL, NULL, NULL);
  pthread_exit(NULL);

  return 0;
}

